src/locales/en.json5

```diff
@@ -507,5 +506,11 @@
   },
   "router": {
     "unsavedChanges": "Are you sure you want to leave this page? Your changes might not be saved."
+  },
+  "formSetting": {
+    "formTech": "Web based form technology",
+    "enableWebForms": "Use WebForms instead of Enketo",
+    "enableWebFormsExplanation": "When users fill out forms via the web (not via ODK Collect), we use Enketo by default. Its successor is \"WebForms\", which is a work in progress that you can already test.",
+    "enableWebFormsExplanationLinktext": "Before enabling this, read up on WebForms and which form features it currently supports!"
```

Comment:
I am thinking to remove these translations and enter static text since they might change